### ISSUE TITLE
Possible feature expansion for concat layer

### ISSUE THREAD

**Issue by snakers4** [nerox8664/pytorch2keras#3]
Today I was struggling with converting my inception-based model.
Apart from the padding hack, I noticed a potential feature for your library.

You assume that only 2 inputs are concatenated into 1, but it case of my model I have 2 and 4. I guess in a more general case you may consider including a proper variation of the below modification into future versions:

```
def convert_concat(node, node_name, input_names, output_name, layers):
    """
    Convert concatenation.

    Args:
        node: pytorch node element.
        node_name: pytorch node name
        input_name: pytorch input node name
        output_name: pytorch output node name
        layers: dictionary with keras tensors
    """
    print('Conerting concat ...')
    
    # basically a very shitty hack because my model contains only 2 or 4 concats
    if len(input_names)==2:
        model0 = layers[input_names[0]]
        model1 = layers[input_names[1]]
        cat = keras.layers.Concatenate(name=output_name, axis=node.dim)
        layers[output_name] = cat([model0, model1])    
    elif len(input_names)==4:
        model0 = layers[input_names[0]]
        model1 = layers[input_names[1]]
        model2 = layers[input_names[2]]
        model3 = layers[input_names[3]]   
        cat = keras.layers.Concatenate(name=output_name, axis=node.dim)
        layers[output_name] = cat([model0, model1, model2, model3])    
```
